src/components/ObjectInfo.ts

```diff
@@ -40,6 +42,24 @@ export default class ObjectInfo extends mixins(MixinUtil) {
       this.data = this.obj;
   }
 
+  async get_meta() {
```

Comment:
getMeta, or something like loadMetaIfNeeded to clarify the purpose of this function